SYCL/DeviceLib/imf_double2half.cpp

```diff
@@ -0,0 +1,60 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// RUN: %clangxx -fsycl -fno-builtin -fsycl-device-lib-jit-link %s -o %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+//
+// UNSUPPORTED: cuda || hip
+
+#include "imf_utils.hpp"
+
+extern "C" {
+_iml_half_internal __imf_double2half(double);
```

Comment:
Is `_iml_half_internal` always supported or should this test also require that the corresponding device has `sycl::aspect::fp16`? Same goes for the other tests.

Reply:
Hi, @steffenlarsen 
Thanks for catching this, already updated the patch.

Thanks very much.